README.md

```diff
@@ -34,7 +34,7 @@ along the way if you don't feel confident to follow those steps yourself
 1. Test your creation within a docker
 
 ```python
-./manage.py docker --model-id MY_MODEL
+./manage.py docker MY_MODEL
```

Comment:
Looks like this flag was deleted, so the model has to be applied without any flag

Reply:
Indeed thanks !